autogen_stubs.sh

```diff
@@ -161,6 +161,17 @@ nv_status_codes = {}
   python3 -c "import tinygrad.runtime.autogen.nv_gpu"
 }
 
+generate_nvcuvid() {
+  clang2py \
+    nvcuvid/cuviddec.h \
```

Comment:
the headers need to be downloaded from the [Nvidia website](https://developer.nvidia.com/nvidia-video-codec-sdk/download) (Video Codec SDK 13.0 Header Files)